internal/oauth/oauth.go

```diff
@@ -47,6 +47,12 @@ var defaultTransport = &http.Transport{
 	ExpectContinueTimeout: expectContinueTimeout,
 }
 
+var validHostnames = map[string]bool{
+	config.NativeHostName:    true,
+	config.ContainerHostName: true,
+	config.ActionsHostName:   true,
```

Comment:
Could you add a comment beside the configs to explain the differences between this hostname?